compile.sh

```diff
@@ -29,5 +32,5 @@ emmake make \
   EXEEXT=.js \
   CFLAGS="-O2 -s EXPORTED_RUNTIME_METHODS=['callMain']"
 
-mkdir -p ../build/
-mv jq.{js,wasm} ../build/
+mkdir -p -- ../build/
+mv -- jq.js jq.wasm ../build/
```

Comment:
Can you undo these 2 lines? The `--` is confusing

Reply:
These are not needed in this specific case with constant paths indeed.